client/components/dashboard/dashboard-directive.js

```diff
@@ -85,6 +85,19 @@ explorer.components.dashboard.DashboardDirective = function() {
       }
 
       /** @export */
+      $scope.getSelectedClass = function(container) {
+        if (container.state().selected) {
+          if (explorerStateService.widgets.selected) {
+            return 'pk-container-selected-partial';
```

Comment:
What's the distinction between a partial and normal container selection? If it's about containers being selected without a widget being selected inside it, that's a bit confusingly named.